static/js/components/PersonalForm.js

```diff
@@ -74,34 +79,17 @@ export default class PersonalForm extends ProfileFormFields {
               {...this.defaultInputComponentProps()}
             />
           </Cell>
+          <Cell col={12}>
+            {this.boundGeosuggest(addressMapping, "Current address",
+              {
+                id: "current-home",
+                placeholder: "Example: 100 Main Street, Anytown, 01234, United States",
+                types: ["geocode"]
```

Comment:
Yup, appears I was looking at a different set of docs for a similar API on a different google service.